kolibri/core/assets/src/vue/content-renderer/index.vue

```diff
@@ -5,7 +5,9 @@
         <h3 class="progress-percent" v-if="progress > 0">
           {{ Math.floor(progress * 100) }}%
         </h3>
-      <div v-el:container></div>
+      <div v-el:container>
+        <loading-spinner></loading-spinner>
```

Comment:
I think for neatness sake, I would rather have a `v-if` on the spinner - even though the loaded plugin does overwrite the DOM element, the loading spinner component is still in memory, so doesn't get nicely cleaned up.